test/parallel/test-v8-global-setter.js

```diff
@@ -26,4 +26,4 @@ require('../common');
 // has a setter interceptor in strict mode.
 // https://github.com/nodejs/node-v0.x-archive/issues/6235
 
-require('vm').runInNewContext('"use strict"; var v = 1; v = 2');
+require('vm').runInNewContext('"use strict"; let v = 1; v = 2');
```

Comment:
In this particular case, the `var` is what is being tested. This file likely should not have been in the task list :-( ... so we won't be able to merge this change but it would be good to still get a PR for you... so maybe find another one of the files to change?